spaces/domains.json

```
@@ -787,6 +787,7 @@
   "vote.cortexdao.io": "cortexdao.eth",
   "gov.emdx.io": "emdx.eth",
   "vote.evoverses.com": "evoverses.games",
+  "vote.wallstreetgames.net": "wsgvote.eth"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "vote.wallstreetgames.net": "wsgvote.eth",
```

---

Fixed